tests/end2end/features/misc.feature

```diff
@@ -671,3 +671,38 @@ Feature: Various utility commands.
         And I wait until the scroll position changed
         And I run :click-element id link
         Then the error "Element position is out of view!" should be shown
+
+    ## :command-history-{prev,next}
+    
+    Scenario: Calling previous command
+        When I run :set-cmd-text :message-info blah
+        And I run :command-accept
+        And I wait for "blah" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-accept
+        Then the message "blah" should be shown
+    
+    Scenario: Browsing through commands 
+        When I run :set-cmd-text :message-info blarg
+        And I run :command-accept
+        And I wait for "blarg" in the log
+        And I run :set-cmd-text :
+        And I run :command-history-prev
+        And I run :command-history-prev
```

Comment:
So this happens because the previous test turns on private browsing and doesn't turn it off!

You can fix it by applying this (e.g. by piping it to `git am`):

```diff
commit 5708bb0306fb28549fc47e686790d006398c27fa
Author: Florian Bruhin <git@the-compiler.org>
Date:   Fri Oct 7 07:44:54 2016 +0200

    Turn off private browsing in misc.feature tests

diff --git a/tests/end2end/features/misc.feature b/tests/end2end/features/misc.feature
index 1cc3683..6829880 100644
--- a/tests/end2end/features/misc.feature
+++ b/tests/end2end/features/misc.feature
@@ -549,6 +549,7 @@ Feature: Various utility commands.
         And I open cookies/set?qute-test=42 without waiting
         And I wait until cookies is loaded
         And I open cookies in a new tab
+        And I set general -> private-browsing to false
         Then the cookie qute-test should be set to 42
 
     ## https://github.com/The-Compiler/qutebrowser/issues/1742
@@ -557,6 +558,7 @@ Feature: Various utility commands.
     Scenario: Private browsing is activated in QtWebKit without restart
         When I set general -> private-browsing to true
         And I open data/javascript/localstorage.html
+        And I set general -> private-browsing to false
         Then the page should contain the plaintext "Local storage status: not working"
 
     Scenario: :repeat-command
```